src/main/webapp/js/instructorCourseDetails.js

```diff
@@ -71,7 +71,13 @@ function attachEventToSendInviteLink() {
         var messageText = 'Usually, there is no need to use this feature because TEAMMATES sends an automatic '
                           + 'invite to students at the opening time of each session. Send a join request anyway?';
         var okCallback = function() {
-            window.location = $clickedLink.attr('href');
+            $.get($clickedLink.attr('href'), function() {
+                var studentEmail = $clickedLink.parent().siblings("td[id|='studentemail']").html().trim();
+                var $message = $('<div class="overflow-auto alert alert-success statusMessage">'
+                              + 'An email has been sent to ' + studentEmail + '</div>');
+                $('#statusMessagesToUser').html($message);
```

Comment:
I'm sorry I didn't understand this. I haven't selected anything twice. I have used the selector only once for retrieving student email. Where exactly can we save the selection?